pkgs/development/python-modules/dm-sonnet/default.nix

```diff
@@ -81,6 +81,7 @@ in buildPythonPackage {
   '';
 
   meta = with lib; {
+    broken = true; # depends on older TensorFlow version than is currently packaged
```

Comment:
broken usually goes last in meta.

Reply:
Good to know, thanks! Amended the `dm-sonnet` / `pymc3` commits accordingly.